<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckEmployeesWindow</class>
 <widget class="QDialog" name="CheckEmployeesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check Employees</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameTextField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Name...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LU ID#:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneTextField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Phone#...</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="idPicture">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="luIdTextField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter LU ID#...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailTextField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Email...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameTextField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Username...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
